storage/blob.go

```diff
@@ -119,6 +119,7 @@ type BlobProperties struct {
 	CopyProgress          string   `xml:"CopyProgress"`
 	CopyCompletionTime    string   `xml:"CopyCompletionTime"`
 	CopyStatusDescription string   `xml:"CopyStatusDescription"`
+	LeaseStatus           string   `xml:"x-ms-lease-status"`
```

Comment:
Please refer to *Response Headers* section in [Get Blob Properties](https://msdn.microsoft.com/en-us/library/azure/dd179394.aspx) for the field : **x-ms-lease-status**.

I'm currently using this property in my app and its working as expected.